crates/madara/node/src/main.rs

```diff
@@ -50,15 +50,32 @@ async fn main() -> anyhow::Result<()> {
         run_cmd.chain_config()?
     };
 
+    // If block time is inferior to the tick time, then only empty blocks will
+    // be produced as we will never update the pending block before storing it.
+    if run_cmd.is_sequencer() && chain_config.block_time < chain_config.pending_block_update_time {
+        tracing::error!(
+            "Block time ({}s) cannot be less than the pending block update time ({}s), as this will yield only empty blocks",
+            chain_config.block_time.as_secs(),
+            chain_config.pending_block_update_time.as_secs()
+        );
+        anyhow::bail!("Block time");
```

Comment:
if we just do panic!() that would be enough right? I mean the tracing::error is covering the message precisely, having Block time here doesn't add any value, wdyt? or we can do: 
```suggestion
        anyhow::bail!(
            "invalid block configuration: block time {} is less than pending update time {}", 
            config.block_time.as_secs(),
            config.pending_block_update_time.as_secs()
        );
```

Reply:
I don't like the use of `panic` as this feels like a correct execution path for the program and not something we should violently abort. The [rust docs](https://doc.rust-lang.org/std/macro.panic.html#when-to-use-panic-vs-result) are a bit unclear regarding this but I feel we shouldn't use it here. Switched the error message to just be `anyhow::bail` since our configuration of `tracing` should print to `stderr` anyway.

Reply:
I see, so you mean developers will be allowed to keep pending tick time > block time?

and yes I agree that we have the same error in the `tracing`, we don't need it in the `anyhow::bail`

Reply:
Yes, well more specifically developers will be _required_ to keep tick time >= block time?

Reply:
you mean block time >= tick time?

Reply:
Oh yes sorry XD